src/combobox/styles/comboBox.m.css

```diff
@@ -4,7 +4,7 @@
 	position: relative;
 	display: inline-block;
 	box-sizing: border-box;
-	width: var(--form-field-width);
+	width: 200px;
```

Comment:
I think the initial sweep of this was just to remove it all, we can do a second sweep to add these in after. This PR keeps getting out of date and is a pain to maintain.